### PR TITLE
Rosetta Code: Dot product

### PR DESCRIPTION
**Pull request by anis79**

### CODE REVIEW

**Comment by privat** [comment]
do not include past commit in the PR, use a new fresh branch

**Comment by ventilooo** [comment]
+1

File: examples/rosettacode/dot_product.nit [hunk 1]
```diff
@@ -0,0 +1,13 @@
+fun dotProduct(v1: Array[Float], v2: Array[Float]): nullable Float
```
Comment (privat) [inline examples/rosettacode/dot_product.nit]: header

**Comment by privat** [comment]
drop useless commits/files from the PR

File: examples/rosettacode/dot_product.nit (hunk 1 above)
Comment (Tagachi) [inline examples/rosettacode/dot_product.nit]: We used snake_case

**Pull request closed without merging**